xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -99,6 +99,8 @@ CVideoPlayerVideo::CVideoPlayerVideo(CDVDClock* pClock
 
 CVideoPlayerVideo::~CVideoPlayerVideo()
 {
+  if (m_pVideoCodec)
+    m_pVideoCodec->Dispose();
```

Comment:
Sorry, I meant the destructor of the codec. Then we can drop Dispose() from the interface of VideoCodec.